core/common/build.gradle.kts

```diff
@@ -53,11 +52,6 @@ android {
             dimension = "flavor"
             isDefault = true
         }
-
-        create("preview") {
```

Comment:
what were these `preview` configs for?

Reply:
They were for creating a version of the app that depended on the pre-release Android V SDK (a.k.a. the "Preview" SDK). Since Android V is now released as Android 15, we no longer need it.

In the future, if we want to add a preview flavor again, we can. I didn't want to leave it in, however, since even AGP is complaining that a lot of the preview features are deprecated. So there may be a new way to do this when the next SDK preview is ready.